app/src/test/java/com/onewelcome/showcaseapp/di/FakeFacadeModule.kt

```diff
@@ -27,4 +30,7 @@ interface FakeFacadeModule {
 
   @Binds
   fun bindPermissionsFacadeFake(permissionsFacadeFake: PermissionsFacadeFake): PermissionsFacade
+
+  @Binds
+  fun bindPreferencesManagerFake(preferencesManagerFake: PreferencesManagerFake): PreferencesManager
```

Comment:
shouldn't this be part of the separate module, smth like "FakeManagerModule"? As currently it's mixed with the FacadeModule

Reply:
now when I look at the changes, maybe we don't need this manager fake at all, as it's injecting the data store. So it should be enough to have fake data store and it will be injected to the manager